### ISSUE TITLE
Settings capacity?

### ISSUE THREAD

**Issue by 3space123** [techyowl/WinkWeb#8]
Possible "Settings" capacity, with locally stored settings cookie(s).  Things like "Zoom Slider" are excellent on big iPad, but really get in the way on small iPhone.  This could later be expanded to support skinning, etc., without having to modify the *.css, *.html and/or *.js file(s).

**Comment by blaineam** [comment]
Settings tab is planned for the future. 

Current settings to include:
F to C
(Encryption Key?)
Hidden tabs/devices

rejected settings options
zoom slider as it will require a user to zoom without seeing the effect of the zoom which is a bad ui practice

**Comment by 3space123** [comment]
For zoom setting, I meant "Show/Hide zoom control", not to preset the zoom.  Does that make sense?  Zoom control great for iPad Pro by the way!

**Comment by blaineam** [comment]
@3space123 
Genius, I totally missed that logic. I thought briefly about showing and hiding the zoom slider but some reason didn't make the connection of putting that button in the settings page. I'm working on that now but I may need help debugging the thermostat scale. I'm done for today though. I added very rough code for the zoom slider hide control and settings tab I'll refine it tomorrow maybe.

**Comment by blaineam** [comment]
supported in version 1.5

**Comment by 3space123** [comment]
You have done such a great job on this project techyowl!  Thanks so much!  You are a pro!

**Comment by blaineam** [comment]
thank you @3space123 but honestly I just fixed a lot of the bugs in the original files from google plus, adding some device support and improving the interface. if I did the project from the start I would have done more with jQuery. to me javascript is simple compared to dealing with other languages I've learned. my biggest project is 23,000+ lines of code and that is a full out custom made social network. [Featheread](https://www.featheread.com) but I'm glad to have developed a project hat more than myself finds useful. Thanks for helping me with the debugging. at least I think it was you. who knows with all these usernames.
